sdcm/tester.py

```diff
@@ -1439,6 +1444,33 @@ def run_stress_thread_bench(self, stress_cmd, duration=None, stress_num=1, keysp
             self.alter_test_tables_encryption(scylla_encryption_options=scylla_encryption_options)
         return bench_thread
 
+    def run_stress_thread_harry(self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='',
+                                # pylint: disable=too-many-arguments,unused-argument
+                                round_robin=False, stats_aggregate_cmds=True, keyspace_name=None,
+                                use_single_loader=False,
+                                stop_test_on_failure=True):  # pylint: disable=too-many-arguments,unused-argument
+
+        timeout = self.get_duration(duration)
+        stop_test_on_failure = False if not self.params.get("stop_test_on_stress_failure") else stop_test_on_failure
```

Comment:
Do you find following code is more clear for this line, don't you?
```python
if not self.params.get("stop_test_on_stress_failure"):
    stop_test_on_failure = False
```

Reply:
Yes